src/Misc.pm

```diff
@@ -1185,12 +1185,14 @@ sub charSelectScreen {
 			}
 		}
 		
-		push @charNames, TF("Slot %d: %s (%s, level %d/%d)%s",
+		push @charNames, TF("Slot %d: %s (%s %s, level %d/%d, %s)%s",
```

Comment:
How does this affect other servers, if at all?

Reply:
my english is not good

but i don't see any problem, as far as i remember,  if server doesn't send mapname, it will show empty string
can you test on your server?

Reply:
![image](https://user-images.githubusercontent.com/17522256/29898906-baa203b0-8dbe-11e7-8255-f2771bba454e.png)

Works with bRO. There should be a comma between Job and Sex,
```push @charNames, TF("Slot %d: %s (%s, %s, level %d/%d, %s)%s",```

Reply:
This is my first time , so how can i do that ? edit my commit ? do i need to make pull again ?